uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Page/AbsencePeriod.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * Class CRM_HRLeaveAndAbsences_Page_AbsencePeriod
+ *
+ * This is the class that handles requests to the Absence Period list and form
+ * pages.
+ */
+class CRM_HRLeaveAndAbsences_Page_AbsencePeriod extends CRM_Core_Page_Basic {
+
+  /**
+   * A list of links available as actions to the items on the page list
+   *
+   * @var array
+   */
+  private $links = array();
+
+  /**
+   * {@inheritdoc}
+   */
+  public function run() {
+    CRM_Utils_System::setTitle(ts('Absence Periods'));
+    parent::run();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function browse() {
+    $object = new CRM_HRLeaveAndAbsences_BAO_AbsencePeriod();
+    $object->orderBy('weight');
+    $object->find();
+    $rows = [];
```

Comment:
either to use array() notation here as other places , or convert any array() usage to short array syntax [] such as in `private $links = array(); ` && `$rows[$object->id] = array();`